webapp/components/generic/SearchableInput/SearchableInput.vue

```diff
@@ -107,15 +107,12 @@ export default {
         this.$emit('query', this.value)
       }, this.delay)
     },
-    /**
-     * TODO: on enter we should go to a dedicated search page!?
-     */
     onEnter(event) {
-      clearTimeout(this.searchProcess)
-      if (!this.loading) {
-        this.previousSearchTerm = this.unprocessedSearchInput
-        this.$emit('query', this.unprocessedSearchInput)
-      }
+      this.$router.push({
+        path: '/search/search-results',
+        query: { search: this.unprocessedSearchInput },
+      })
+      this.$emit('clearSearch')
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-09-23T08:40:08Z" title="Wednesday, September 23rd 2020, 10:40:08 am +02:00">Sep 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -107,15 +107,12 @@ export default {
         this.$emit('query', this.value)
       }, this.delay)
     },
-    /**
-     * TODO: on enter we should go to a dedicated search page!?
-     */
     onEnter(event) {
-      clearTimeout(this.searchProcess)
-      if (!this.loading) {
-        this.previousSearchTerm = this.unprocessedSearchInput
-        this.$emit('query', this.unprocessedSearchInput)
-      }
+      this.$router.push({
+        path: '/search/search-results',
+        query: { search: this.unprocessedSearchInput },
+      })
+      this.$emit('clearSearch')
```


If you hit enter, it clears the menu under the search field. But the field is not empty. The menu says then `Nothing found`. That’s a bit misleading, because the results are just underneath on the search page.